lntest/node.go

```diff
@@ -307,11 +350,18 @@ func (hn *HarnessNode) start(lndError chan<- error) error {
 
 		// Signal any onlookers that this process has exited.
 		close(hn.processExit)
+
+		// Make sure log file is closed and renamed if necessary.
+		hn.finalizeLogfile()
 	}()
 
 	// Write process ID to a file.
 	if err := hn.writePidFile(); err != nil {
 		hn.cmd.Process.Kill()
+
+		// Make sure log file is closed and renamed if necessary.
+		hn.finalizeLogfile()
```

Comment:
Since we kill the process, I think maybe this will be called from the goroutine regardless?